src/openrct2/scripting/bindings/ride/ScRide.cpp

```diff
@@ -564,6 +630,17 @@ namespace OpenRCT2::Scripting
         dukglue_register_property(ctx, &ScRide::maxLiftHillSpeed_get, nullptr, "maxLiftHillSpeed");
         dukglue_register_property(ctx, &ScRide::minLiftHillSpeed_get, nullptr, "minLiftHillSpeed");
         dukglue_register_property(ctx, &ScRide::satisfaction_get, nullptr, "satisfaction");
+        dukglue_register_property(ctx, &ScRide::maxSpeed_get, nullptr, "maxSpeed");
+        dukglue_register_property(ctx, &ScRide::averageSpeed_get, nullptr, "averageSpeed");
+        dukglue_register_property(ctx, &ScRide::rideTime_get, nullptr, "rideTime");
+        dukglue_register_property(ctx, &ScRide::rideLength_get, nullptr, "rideLength");
+        dukglue_register_property(ctx, &ScRide::maxPositiveVerticalGs_get, nullptr, "maxPositiveVerticalGs");
+        dukglue_register_property(ctx, &ScRide::maxNegativeVerticalGs_get, nullptr, "maxNegativeVerticalGs");
+        dukglue_register_property(ctx, &ScRide::maxLateralGs_get, nullptr, "maxLateralGs");
+        dukglue_register_property(ctx, &ScRide::totalAirTime_get, nullptr, "totalAirTime");
+        dukglue_register_property(ctx, &ScRide::drops_get, nullptr, "drops");
+        dukglue_register_property(ctx, &ScRide::poweredLifts_get, nullptr, "poweredLifts");
```

Comment:
Sorry for this late addition, but:
1. Should we add `num` before these two, to make clear they are counts?
2. Digging through the code, it looks like the second property ultimately represent the number of lifts hill, whether LIM Launched or conventional chain lift. So I would propose just calling this `liftHills` or `numLiftHills`, depending on what you picked for point 1.

Reply:
Done